src/plugins/Transloadit/Client.js

```diff
@@ -0,0 +1,51 @@
+module.exports = class Client {
+  constructor (opts) {
+    this.apiUrl = 'https://api2.transloadit.com'
+    this.opts = opts
+  }
+
+  /**
+   * Create a new assembly.
+   *
+   * @param {object} options
+   */
+  createAssembly ({ templateId, expectedFiles }) {
+    const data = new FormData()
+    data.append('params', JSON.stringify({
+      template_id: templateId,
+      auth: { key: this.opts.key }
+    }))
+    data.append('fields', JSON.stringify({
+      // Nothing yet.
+    }))
+    data.append('tus_num_expected_upload_files', expectedFiles)
+
+    return fetch(`${this.apiUrl}/assemblies`, {
+      method: 'post',
+      body: data
+    }).then((response) => response.json()).then((assembly) => {
+      if (assembly.error) {
+        const error = new Error(assembly.message)
+        error.code = assembly.error
+        error.status = assembly
+        throw error
+      }
+
+      return this.getAssemblyStatus(assembly.status_endpoint)
+    })
+  }
+
+  /**
+   * Get the current status for an assembly.
+   *
+   * @param {string} url The status endpoint of the assembly.
+   */
+  getAssemblyStatus (url) {
+    return fetch(url)
+      .then((response) => response.json())
+  }
+
+  getTusEndpoint () {
+    return `${this.apiUrl}/resumable/files/`
```

Comment:
This was added now ❤️  See for example: https://api2-cassidy.transloadit.com/assemblies/f0a8d2a01e8c11e784f5a5dd004c11d1

![screen shot 2017-04-11 at 10 01 13](https://cloud.githubusercontent.com/assets/26752/24898624/db6facfe-1e9d-11e7-8f2b-13527e3a1ea2.png)